incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -315,6 +316,69 @@ Similarly, if you want to make `ns-bar` a root again, you must be an
 administrator of the root namespace that is an ancestor of `ns-bar`, since the
 admins of that namespace will lose access to `ns-bar` once it becomes a root.
 
+
+<a name="move-subnamespace"/>
+
+### Move a Subnamespace
+
+Changing the parent of a subnamespace is not yet permitted in HNC, though there may be times
+where you want to move a subnamespace from one parent to another. 
```

Comment:
Let's use the language of "conversion" here - e.g., how to convert a subns to a full namespace and back.